resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -64,3 +64,24 @@ function createListElement(date, type, text, id) {
 
   return liElement;
 }
+
+/**
+ * Fetches the blobstore-serve to sends its response as an array buffer to the Adobe DC View
+ */
+async function getRevieweeResume() {
+  fetch('/blobstore-serve')
+    .then((response) => {
+      var adobeDCView = new AdobeDC.View({
+        clientId: "b98bbf69d44442479396583253ac267c",
```

Comment:
I actually think it's okay to leave this client ID in GitHub instead of putting it in a private storage location. It looks like Adobe checks to make sure the client ID is being used from the registered domain (in this case, the deployed ResumeBuddy URL), so a random person won't be able to use the client ID without access to deploying on the ResumeBuddy domain.